samba/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: samba_homepage_1
+      APP_PORT: 9445
+    container_name: samba_homepage_app_proxy_1
+
+  apache:
+    image: >-
+      httpd:2.4.59@sha256:10182d88d7fbc5161ae0f6f758cba7adc56d4aae2dc950e51d72c0cf68967cea
+    container_name: samba_homepage_1
+    restart: on-failure
+    volumes:
+      - ${APP_DATA_DIR}/data:/usr/local/apache2/htdocs
+      - ${APP_DATA_DIR}/data/httpd.conf:/usr/local/apache2/conf/httpd.conf
```

Comment:
For just hosting a static web page, Apache httpd seems a bit heavy to me. Maybe consider alternatives like [Static Web Server](https://static-web-server.net/).
The image size of `httpd` is 220MB, while the image size of `joseluisq/static-web-server` is 15MB, therefore I think it it is really worth changeing it.